packages/runtime/src/stratus.ts

```diff
@@ -1441,25 +1448,42 @@ Stratus.Internals.LoadImage = (obj: any) => {
                     jQuery(this).remove() // prevent memory leaks
                 })
                 loadEl.on('error', () => {
-                    // TODO: Go down in sizes before reaching the origin
-                    // Standardize src
+                    // This is the smallest size, so there is nothing to fallback to. Let's hope a bigger size below fixes the image
                     el.attr('data-loading', dehydrate(false))
-                    el.css('background-image', 'url(' + srcOriginProtocol + ')')
                     if (cookie('env')) {
-                        console.log('Unable to load', size.toUpperCase(), 'size.', 'Restored:', srcOriginProtocol)
+                        console.warn('LoadImage() Unable to load', size.toUpperCase(), 'size at ', srcOriginProtocol)
                     }
+                    jQuery(this).remove() // prevent memory leaks
                 })
             }
 
+            // Set up actions to preload and replace the small image with the desire size.
+            // onLoad and onError (if size doesn't exist, just don't use the prefetched image)
             // Change the Source to be the desired path (for image or background)
             const srcProtocol: any = src.startsWith('//') ? window.location.protocol + src : src
             el.attr('data-loading', dehydrate(false))
             el.attr('data-size', dehydrate(size))
-            if (type === 'img') {
-                el.attr('src', srcProtocol)
-            } else {
-                el.css('background-image', 'url(' + srcProtocol + ')')
-            }
+
+            // Preload this image first. Ensures speed to display and image is valid
```

Comment:
This is far better logic, as we will ensure the image exists before we replace them.  Well done.